roles/pulp-api/tasks/get_node_ip.yml

```diff
@@ -115,7 +115,7 @@
 
 - name: Set token_server and other container default settings
   set_fact:
-    token_server_dict: "{ 'token_server': '{{ token_server }}', 'token_auth_disabled': 'False', 'token_signature_algorithm': 'ES256','public_key_path': '/etc/pulp/keys/container_auth_public_key.pem', 'private_key_path': '/etc/pulp/keys/container_auth_private_key.pem', 'galaxy_feature_flags': { 'execution_environments': 'True' }  }"
```

Comment:
Just for the sake of curiosity, there are 2 ways to avoid the problem if writing to this variable is needed.

1. traversing with `__`

```yaml
'galaxy_feature_flags__execution_environments': 'true'
```

2. setting the merge

```yaml
'galaxy_feature_flags': { 'execution_environments': 'true' , 'dynaconf_merge': 'true'}
```

Also notice that 'True' and 'False'  will be parsed as string on the final django settings, if the intention is a boolean then values must be in `true, false, on, off, enabled, disabled'